src/library/dao/trackdao.cpp

```diff
@@ -1737,14 +1737,34 @@ bool TrackDAO::updateTrack(const Track& track) const {
     return true;
 }
 
+// Make sure that `directory` in in track_locations table is indeed a
```

Comment:
can you add a link to the github issue so people can follow the discussion if they come across this code?

Reply:
:+1: